cgroup/cgroup_darwin.go

```diff
@@ -16,34 +17,34 @@ type cgroup struct {
 
 // Read reads the given cgroup file data and returns the content as a string
 func (cg cgroup) Read(controller, file string) (string, error) {
-	return "", nil
+	return "", fmt.Errorf("not implemented")
```

Comment:
I thought we wanted "not supported" everywhere in this file?

Reply:
I personally have no opinion on the wording, as long as we throw an error with a relevant message. I'm happy to change it if it makes more sense though.

Reply:
no one should ever see this error, so it doesn't matter